app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -24,6 +24,7 @@ def initialize(ems, options = nil)
         @rgs               = resource_group_service(@config)
         @sas               = storage_account_service(@config)
         @mis               = managed_image_service(@config)
+        @vmis              = virtual_machine_image_service(@config, :location => @ems.provider_region)
```

Comment:
Ah, right, I forgot the constructor for VirtualMachineImageService is a little different and accepts a location argument.